### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by ANJALI-VANGURI**
Hello @Vishwajith-Shettigar ,
 
To enhance project structure, added project structure section, detail pull request process, and clarify issue report process. I hope you like the changes that I have done.
 please merge the pull request.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,39 +1,152 @@
...
+## 7. Good Coding Practices 📝
+- Write clean, readable code.
+- Use meaningful names and comment where needed.
 
----
 
-Thank you for contributing to the neXtgen project! 🎉
+<br>
+
+# Need Help with the Basics? 🤔
```
Comment (Vishwajith-Shettigar) [inline CONTRIBUTORS.md]: _

> _# Need Help with the Basics? 🤔_

_
Replace with this
# Need help getting started?

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1,39 +1,152 @@
...
+
+<br>
+
+
+
+# Pull Request Process 🚀
+
+When submitting a pull request, please adhere to the following:
+
+1. **Self-review your code** before submission. 😀
```
Comment (Vishwajith-Shettigar) [inline CONTRIBUTORS.md]: Remove the emoji.

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -1,39 +1,152 @@
...
+
+<br>
+
+# Issue Report Process 📌
+
+To report an issue, follow these steps:
+
+1. Navigate to the project's issues section :- [Issues](https://github.com/Vishwajith-Shettigar/NextGen/issues)
+2. Provide a clear and concise description of the issue.
+3. **Avoid spamming** to claim an issue. Patience is key! 😀
```
Comment (Vishwajith-Shettigar) [inline CONTRIBUTORS.md]: Remove the emoji.

**Review by Vishwajith-Shettigar — CHANGES_REQUESTED** [review]
Thanks @a-n-u-vanguri, i have left some review comments, PTAL.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (ANJALI-VANGURI) [reply CONTRIBUTORS.md]: OK Sure.

File: CONTRIBUTORS.md (hunk 2 above)
Comment (ANJALI-VANGURI) [reply CONTRIBUTORS.md]: Sure.

File: CONTRIBUTORS.md (hunk 3 above)
Comment (ANJALI-VANGURI) [reply CONTRIBUTORS.md]: OK.

**Comment by ANJALI-VANGURI** [comment]
Hi! @Vishwajith-Shettigar ,

I've made the requested changes. Could you please merge the pull request now? Thank you!

**Pull request merged**